source/basic.tex

```diff
@@ -2233,8 +2233,8 @@
 
 \pnum
 \indextext{program}%
-A \defn{program} consists of one or more \defn{translation
-units} (Clause~\ref{lex}) linked together. A translation unit consists
+A \defn{program} consists of one or more \defn{translation units}
```

Comment:
The usual solution would be `\defnx{plural}{singular}`. We've done that or similar things elsewhere. I think that'd be appropriate here.